xlcalculator/ast_nodes.py

```diff
@@ -47,10 +47,15 @@ def __init__(self, namespace=None, ref=None, seen=None):
         self.seen = seen if seen is not None else []
         self.namespace = namespace if namespace is not None else xl.FUNCTIONS
         self.ref = ref
+        self.refsheet = ref.split("!")[0]
+        self.sheet = self.refsheet
```

Comment:
Combine into single, multi-assginment statement.

Reply:
Add new attributes in class declaration above. The usage of `refsheet` and `sheet` is exactely the opposite of what I would have thought, so it needs some documentation (at the class level).

Reply:
From what I understood ref always pointed towards the original cell we're evaluating?